Dockerfile

```diff
@@ -6,15 +6,15 @@ LABEL maintainer="Marko Kajzer <markokajzer91@gmail.com>"
 RUN apk add --no-cache --quiet build-base ffmpeg git make python
 
 WORKDIR /app
-COPY package*.json ./
-RUN npm install --quiet
+COPY package*.json yarn.lock ./
```

Comment:
Since we are touching this line, could you make this `package.json`, instead of `package*.json`?